src/js/api.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
```

Comment:
- [ ] You should solve linter issues instead of suppressing the warnings. Please remove all `/* eslint-disable */` comments